### PR TITLE
sort PIDs

### PR DESCRIPTION
**Pull request by dibyaaaaax**

### CODE REVIEW

File: mwtext/utilities/preprocess_wikidata.py [hunk 1]
```diff
@@ -37,7 +37,16 @@
...
 def preprocess_wikidata(dump, verbose=False):
+
+    def PID_comparator(claim_tuple):
+        try:
+            return sorted_PIDs.index(claim_tuple[0])
+        except:
+            return len(sorted_PIDs)+1
+
     wikidata_preprocessor = WikidataPreprocessor()
+    sorted_PIDs = wikidata_preprocessor.get_sorted_properties()
```
Comment (dibyaaaaax) [inline mwtext/utilities/preprocess_wikidata.py]: I created the comparator in here just to keep things similar to `preprocess_wikitext`. The other option would be to move lines 47 and 48 out in the global scope too. What do you think?

**Comment by dibyaaaaax** [comment]
@geohci I have restructured WikidataPreprocessor class as discussed in the other [PR](https://github.com/mediawiki-utilities/python-mwtext/pull/11). This will probably make testing for the sort method much easier. What do you think?

**Comment by halfak** [comment]
Thanks for your work on this.  It looks like we'll need to fold this into a recent refactoring of the library.  I'm working on a follow-up commit that will do exactly that.

**Comment by halfak** [comment]
I'd like to close this in favor of https://github.com/mediawiki-utilities/python-mwtext/pull/15 

That PR adjust the structure to reflect the recent refactoring.  @dibyaaaaax, could you take a look at that work to see if it captures your thoughts from this PR correctly?

**Comment by dibyaaaaax** [comment]
it does! Thanks for the adjustments :)

**Pull request closed without merging**